internal/incoming/odfi/processor.go

```diff
@@ -120,19 +120,19 @@ func processDir(dir string, alerters alerting.Alerters, auditSaver *AuditSaver,
 	return el
 }
 
-func processFile(path string, alerters alerting.Alerters, auditSaver *AuditSaver, fileProcessors Processors) error {
+func processFile(path string, alerters alerting.Alerters, auditSaver *AuditSaver, validation ach.ValidateOpts, fileProcessors Processors) error {
 	bs, err := os.ReadFile(path)
 	if err != nil {
 		return fmt.Errorf("problem opening %s: %v", path, err)
 	}
 	bs = bytes.TrimSpace(bs)
 
 	reader := ach.NewReader(bytes.NewReader(bs))
-	reader.SetValidation(&ach.ValidateOpts{
-		AllowMissingFileControl:    true,
-		AllowMissingFileHeader:     true,
-		AllowUnorderedBatchNumbers: true,
-	})
+	// Enable some default ACH ValidateOpts
+	validation.AllowMissingFileControl = true
```

Comment:
Should these respect the passed in opts of defined already?

Reply:
Maybe? They've been the defaults for quite some time, so I'm hesitant to require that they are set now.